spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -100,6 +103,11 @@
 ACTIVATION_KEYS={activation_keys}
 ORG_GPG_KEY={org_gpg_key}
 
+# To reactivate single Salt client use following variable:
+# NOTE: Reactivation keys are removed valid only for single use.
+#       It is also possible to use REACTIVATION_KEY environmental variable.
+REACTIVATION_KEY=${{REACTIVATION_KEY:-}}
```

Comment:
Please adapt also the code for traditional registration.
When a customer use this also for tranditonal registration, the reactivation key will not be send as it is ignored.
Just concat 
```
  REACTIVATION_KEY,ACTIVATION_KEYS
```

Reply:
Added at line 976.